docs/source/x_ext.rst

```diff
@@ -400,7 +408,7 @@ then a new compressed request transaction started).
   +------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------+
   | Signal                 | Type                 | Description                                                                                                     |
   +========================+======================+=================================================================================================================+
-  | ``instr``              | logic [31:0]         | Uncompressed instruction.                                                                                       |
+  | ``instr``              | logic [31:0]         | Uncompressed instruction. Must be signalled as 0 if ``accept`` is 0.                                            |
```

Comment:
similar statements are already in place for all fields of the issue_resp. I think it would be good to have it consistent. Might be a topic to discuss next wednesday. I will forward you the invite to the call

Reply:
Sorry I didn't catch this on ``issue_resp`` during my initial review.
Just a general comment from my side:
- OK for consistent behaviour on all interface
- policy enforcement should be avoided in a specification if alternative solutions may exist to solve the same problem.

My understanding here is that the proposed policy is linked to a particular solution (OR-combining response signal) when connecting multiple coprocessors to the same interface. Since a bit of glue logic is required to do that, it is up to the glue logic to solve the problem.

Reply:
I don't fully agree that there is a consistency requirement w.r.t. forcing signals to 0. In the issue interface for example writeback = 1 is defined as 'accepted plus writeback required'. I would be against forcing all kind of signals to 0 for consistency. If you really want consistency then I would rather remove the forcing to 0 on the issue interface (and other interfaces if it exists).